fetch/connection-pool/resources/network-partition-key.py

```diff
@@ -85,29 +83,29 @@ def handle_fetch_file(request, response, partition_id, uuid):
     if not subresource_origin or not rel_path or not include_credentials:
         return simple_response(request, response, 404, b"Not found", b"Invalid query parameters")
 
-    cur_path = os.path.realpath(__file__)
+    cur_path = os.path.realpath(isomorphic_decode(__file__))
     base_path = os.path.abspath(os.path.join(os.path.dirname(cur_path), os.pardir, os.pardir, os.pardir))
-    path = os.path.abspath(os.path.join(base_path, rel_path))
+    path = os.path.abspath(os.path.join(base_path, isomorphic_decode(rel_path)))
 
     # Basic security check.
     if not path.startswith(base_path):
         return simple_response(request, response, 404, b"Not found", b"Invalid path")
 
     sandbox = request.GET.first(b"sandbox", None)
-    if sandbox == "true":
-        response.headers.set(b"Content-Security-Policy", "sandbox allow-scripts")
+    if sandbox == b"true":
+        response.headers.set(b"Content-Security-Policy", b"sandbox allow-scripts")
 
-    file = open(path, mode="r")
+    file = open(path, mode="rb")
     body = file.read()
     file.close()
 
-    subresource_path = "/" + os.path.relpath(__file__, base_path).replace('\\','/')
-    subresource_params = "?partition_id=" + partition_id + "&uuid=" + uuid + "&subresource_origin=" + subresource_origin + "&include_credentials=" + include_credentials
-    body = body.replace("SUBRESOURCE_PREFIX:", subresource_origin + subresource_path + subresource_params)
+    subresource_path = u"/" + os.path.relpath(isomorphic_decode(__file__), base_path).replace(u'\\', u'/')
```

Comment:
Nit: it's better to make `subresource_path` binary in the first place, as all subsequent uses (along with `subresource_params`) are all binary.

Reply:
Code updated. Thank you!